diff --git a/node_modules/hafas-client/parse/arrival-or-departure.js b/node_modules/hafas-client/parse/arrival-or-departure.js
index b0ea6ef..477193b 100644
--- a/node_modules/hafas-client/parse/arrival-or-departure.js
+++ b/node_modules/hafas-client/parse/arrival-or-departure.js
@@ -36,6 +36,7 @@ const createParseArrOrDep = (prefix) => {
 			remarks: [],
 			origin: null,
 			destination: null,
+			directionFlag: d.dirFlg || null,
 		};
 
 		if (prefix === DEPARTURE && Array.isArray(d.prodL) && d.prodL[0] && locL[d.prodL[0].tLocX]) {
